lms/static/js/spec/student_account/register_spec.js

```diff
@@ -366,6 +369,43 @@
                 // Form button should be disabled on success.
                     expect(view.$submitButton).toHaveAttr('disabled');
                 });
+
+                it('displays a modal with the terms of service', function() {
+                    createRegisterView(this);
+
+                // Check there is no modal container initially
```

Comment:
Yeah. It's an existing convention in that spec file (it got my attention too). See the [comments in the test for registering a user]( https://github.com/edx/edx-platform/blob/master/lms/static/js/spec/student_account/register_spec.js#L250-L271).